easybuild/easyconfigs/p/pyfaidx/pyfaidx-0.8.1.3-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
```

Comment:
Please also exclude this extra version so that it isn't used by mistake and create conflicts.

`eb --update-pr :pyfaidx-0.8.1.3-GCCcore-13.3.0.eb`